app/views/static/english_language_guidance.html.erb

```diff
@@ -0,0 +1,91 @@
+
+<p class="govuk-body">
+  Each provider offers a number of different tests, so it’s important to understand which ones we can accept.
+</p>
+
+<p class="govuk-body">
+  You’ll also need to know your provider’s required scores for the 4 disciplines of listening, speaking, reading and writing.
+</p>
+
+<%= govuk_table do |table| %>
+  <% table.with_caption(size: 'm', text: 'Approved providers and tests') %>
+
+  <% table.with_head do |head| %>
+    <% head.with_row do |row| %>
+      <% row.with_cell(text: 'Approved provider name') %>
+      <% row.with_cell(text: 'Tests we can accept') %>
+      <% row.with_cell(text: 'Required scores') %>
+    <% end %>
+  <% end %>
+
+  <% table.with_body do |body| %>
+    <% EnglishLanguageProvider.all.each do |provider| %>
+      <% body.with_row do |row| %>
+        <% row.with_cell do %>
+          <%= govuk_link_to provider.name, provider.url%>
+        <% end %>
+        <% row.with_cell(text: provider.accepted_tests) %>
+        <% row.with_cell do %>
+          Listening:&nbsp;<%= provider.b2_level_requirement %><br>
+          Speaking:&nbsp;<%= provider.b2_level_requirement %><br>
+          Reading:&nbsp;<%= provider.b2_level_requirement %><br>
+          Writing:&nbsp;<%= provider.b2_level_requirement %><br>
+        <% end %>
+      <% end %>
+    <% end %>
+  <% end %>
+<% end %>
+
+
+<%= govuk_warning_text(icon_fallback_text: "Warning") do %>If you’re already in the UK when you book your SELT, you
+  cannot use PSI Services (UK) Ltd. If you’re outside the UK, you cannot use Trinity College London.
+<% end %>
+
+<h4 class="govuk-heading-m">Deciding which test is right for you</h4>
+
+<p class="govuk-body">We cannot recommend which of the approved providers is right for you. Prices and locations vary between providers, so you’ll need to research costs, as well as the availability of an approved test location near you.
+
+</p><h5 class="govuk-heading-m">About other SELT providers</h5>
+
+<p class="govuk-body">The TRA does not recognise any other providers or SELTS at the moment. If they do so in future, those providers and tests will also be listed on this page.
+
+</p><p class="govuk-body">If you plan to use your SELT for a Skilled Worker visa, you must use one of the 5 approved providers listed on this page. No future providers or tests will be accepted for this purpose.
+
+</p><h6 class="govuk-heading-m">Providing evidence of your SELT</h6>
+
+<p class="govuk-body">The QTS application form will ask you to provide the reference number for your SELT. The approved providers use different names for these numbers. These are:</p>
+
+<ul class="govuk-list govuk-list--bullet">
+    <li>IELTS SELT Consortium – ‘UKVI number’</li>
+    <li>LanguageCert – ‘Candidate URN’</li>
+    <li>Pearson – ‘SELT URN’</li>
+    <li>PSI Services (UK) Ltd – ‘URN’</li>
+    <li>Trinity College London – ‘UER’</li>
+</ul>
```

Comment:
Just spotted this another list where we're hard coding the list of providers but we'll want to get this from the database instead - it might require another field.